packages/contents/src/contents.ts

```diff
@@ -589,11 +589,14 @@ export class Contents implements IContents {
             mimetype: mimetype || 'text/plain',
           };
         } else {
+          const byteToString = (data: string, byte: number) =>
```

Comment:
Sorry, was on my phone earlier: i meant hoist _outside_ the function (or even the class) so that we're not defining a fresh function/scope every time we use this. BUT: turns out we _already_ define a bunch of anonymous functions all over this file, so it's unfair to ask for micro-optimizations for new contributions. I've raised #53, so let's not worry about it here!